app/views/jobseekers/accounts/show.html.haml

```diff
@@ -19,7 +19,7 @@
         %dt.govuk-summary-list__key= t(".summary_list.password")
         %dd.govuk-summary-list__value= t(".summary_list.password_placeholder")
         %dd.govuk-summary-list__actions
-          = govuk_link_to edit_jobseeker_registration_path do
+          = govuk_link_to edit_jobseeker_registration_path(update_password: true) do
```

Comment:
What do you think about setting `to_update: 'password'` or `to_update: 'email'`, instead of having email as the default and password as the exception?

It would be more verbose in the controller, but more explicit in the view (as there isn't a reason for email to be the default field to update except for historical contingency - the email part was written first).

```ruby
class Jobseekers::RegistrationsController < Devise::RegistrationsController
  def updating_password?
    params[:to_update] == "password" || params[:commit] == t("buttons.update_password")
  end
  
  def updating_email?
    params[:to_update] == "email" || params[:commit] == t("whatever")
  end  
end
```

Reply:
Or, with a little edit to the .yml, in the view you could have e.g.
```ruby
t(".#{ @to_update }.title")
```
Instead of various conditionals.

Reply:
It is clear enough to me what this code is doing and being more explicit and more verbose do not add much here. But that is just my opinion